docs/markdown/Python/python-integrations/google-cloud-function-python.md

```diff
@@ -88,4 +88,4 @@ Step 4: Upload to Google Cloud
 
 You can use any of the various Google Cloud methods to upload your zip file, such as the Google Cloud console or the [Google Cloud CLI](https://cloud.google.com/functions/docs/deploying/filesystem#deploy_using_the_gcloud_tool).
 
-You must specify the handler as `main.handler`.
+You must specify the handler as `handler`.
```

Comment:
The "For example" section above still says `main.handler`: assuming that that is supposed to be aligned here, it might be good for this line to say something like "as demonstrated in the example above."

Reply:
I'm not really following your suggestion, but the "For example" output was no longer accurate. I fixed that just like I was forced to fix it in the tests.